addons/website_slides/models/slide_slide.py

```diff
@@ -603,6 +603,15 @@ def _compute_can_publish(self):
     def _get_can_publish_error_message(self):
         return _("Publishing is restricted to the responsible of training courses or members of the publisher group for documentation courses")
 
+    @api.constrains('question_ids')
+    def _check_question_has_answers(self):
```

Comment:
I've noticed this constraint is only useful in one case: a quiz containing only empty question(s) (with no answers). 
In other cases, the constraint on slide_question takes empty questions into account.
I wonder if it worth adding it, regarding its very limited/rare use.

Reply:
Isn't it used if you add a question without adding answers, even if there are already other valid questions?

Reply:
That's right. If you add an empty question to an existing quiz (containing valid questions already saved) and save, the validation error comes from the slide constraint (slide_slide.py).